lib/trino-parquet/src/main/java/io/trino/parquet/writer/PrimitiveColumnWriter.java

```diff
@@ -20,6 +20,7 @@
 import io.trino.parquet.writer.repdef.RepLevelIterable;
 import io.trino.parquet.writer.repdef.RepLevelIterables;
 import io.trino.parquet.writer.valuewriter.PrimitiveValueWriter;
+import io.trino.spi.block.Block;
```

Comment:
Is it possible to add a test?

Reply:
`ParquetTester#testRoundTrip` already tests with and without nulls
`AbstractTestParquetReader` also has tests with flat as well as nested types